independent-publisher-connectors/COVID-19/apiDefinition.swagger.json

```diff
@@ -0,0 +1,534 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "COVID-19",
```

Comment:
@last72 As the word "COVID-19" is generic, it can't be used for a connector title. You can name your connector with "Covid-19 data by the JHU CSSE" or "JHU CSSE Covid-19" or "COVID-19 JHU CSSE" or similar titles. You may need to update your connector folder name accordingly.

Reply:
Changed to "COVID-19 JHU CSSE"